src/Gelf/Logger.php

```diff
@@ -181,7 +181,7 @@ private static function interpolate($message, array $context)
         // build a replacement array with braces around the context keys
         $replace = array();
         foreach ($context as $key => $val) {
-            $replace['{' . $key . '}'] = $val;
+            $replace['{' . $key . '}'] = is_string($val) ? $val : 'Array';
```

Comment:
This implementation has a note: "The following is an example implementation of placeholder interpolation provided for reference purposes only".
Probably a more valid interpolation has been done in the monolog project: https://github.com/Seldaek/monolog/blob/master/src/Monolog/Processor/PsrLogMessageProcessor.php